packages/rpc-provider/src/http/send.spec.ts

```diff
@@ -32,7 +34,7 @@ describe('send', () => {
     return http
       .send('test_body', ['param'])
       .then((result) => {
-        expect(mock.body['test_body']).toEqual({
+        expect(Object.values(mock.body)[0]).toEqual({
```

Comment:
I believe test_body is easier to read

Reply:
`Element implicitly has an 'any' type because type 'Object' has no index signature.`
`mock.body['test_body']`

Reply:
`(mock.body as any)['test_body']` is working.